ALUM/ALUM/Views/SignUpPage/SignUpPageViewModel.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  SignUpPageViewModel.swift
+//  ALUM
+//
+//  Created by Neelam Gurnani on 2/17/23.
+//
+
+import Foundation
+import FirebaseCore
+import FirebaseAuth
+
+final class SignUpPageViewModel: ObservableObject {
+    @Published var passwordAgain: String = ""
+    @Published var disabled: Bool = true
```

Comment:
action - where is this used? if this disables some button, let's make the variable name more explicit to say disableButton. Change the name according to what it disabled